Developer-Guide.md

```diff
@@ -277,10 +286,131 @@ To perform analysis on Kata logs, use the
 [`kata-log-parser`](https://github.com/kata-containers/tests/tree/master/cmd/log-parser)
 tool.
 
+See also [Setting up a debug console](#setting-up-a-debug-console).
+
 # Appendices
 
 ## Checking Docker default runtime
 
 ```
 $ sudo docker info 2>/dev/null | grep -i "default runtime" | cut -d: -f2- | grep -q runc  && echo "SUCCESS" || echo "ERROR: Incorrect default Docker runtime"
 ```
+
+## Setting up a debug console
+
+By default, it is not possible to login to a virtual machine since this could
```

Comment:
Lines 301-304 suggested rewrite (unless I've changed the meaning): By default you cannot login to a virtual machine since this can be sensitive from a security perspective. However a virtual machine also requires additional packages in the rootfs, which means the image used to boot each virtual machine is larger.

Reply:
After a coffee, that paragraph reads rather strangely in both forms :)

I've tried to reword using your suggested phrasing and come up with:

> By default you cannot login to a virtual machine since this can be sensitive
from a security perspective. Also allowing logins would require additional
packages in the rootfs, which would increase the size of the image used to
boot the virtual machine.

Reply:
Ha! I like your phrasing. Thanks :)